src/java.desktop/share/classes/java/awt/Robot.java

```diff
@@ -145,15 +146,15 @@ public Robot() throws AWTException {
      * If screen devices are reconfigured such that the coordinate system is
      * affected, the behavior of existing Robot objects is undefined.
      *
-     * @param screen    A screen GraphicsDevice indicating the coordinate
-     *                  system the Robot will operate in.
-     * @throws  AWTException if the platform configuration does not allow
-     * low-level input control.  This exception is always thrown when
-     * GraphicsEnvironment.isHeadless() returns true.
-     * @throws  IllegalArgumentException if {@code screen} is not a screen
-     *          GraphicsDevice.
-     * @see     java.awt.GraphicsEnvironment#isHeadless
-     * @see     GraphicsDevice
+     * @param screen A screen GraphicsDevice indicating the coordinate
+     *               system the Robot will operate in.
+     * @throws AWTException             if the platform configuration does not allow
+     *                                  low-level input control.  This exception is always thrown when
+     *                                  GraphicsEnvironment.isHeadless() returns true.
+     * @throws IllegalArgumentException if {@code screen} is not a screen
+     *                                  GraphicsDevice.
+     * @see java.awt.GraphicsEnvironment#isHeadless
+     * @see GraphicsDevice
```

Comment:
It's exactly the same here: the formatting is changed, yet nothing's touched in this method.

You should avoid changing or reformatting lines of code which you don't touch… unless the purpose of a PR is to update formatting and perform some clean-ups.